docs/_templates/search.html

```diff
@@ -32,13 +50,92 @@
                 query,
                 {
                     hitsPerPage: 50,
+                    page: page,
                     facetFilters: [getVersionFacetFilter()]
                 }
             )
-                .then(({ hits }) => addResultsList(hits, query, searchResultsContainer))
+                .then(result => {
+                // Display pagination if more than 1 page returned
+                let nbPages = result["nbPages"]
+                if (nbPages > 1) {
+                    document.querySelector("#pagination").removeAttribute("hidden")
+                    displayPagination(page, nbPages)
+                }
+
+                // Display hits
+                let hits = result["hits"]
+                addResultsList(hits, query, searchResultsContainer)
+            })
                 .catch((error) => console.log(error))
         }
 
+        function displayPagination(page, totalPages) {
+            const paginationDiv = document.querySelector("#pagination")
+            const paginationList = paginationDiv.querySelector(".pagination-list")
+
+            // Hide previous/next button if showing first/last page
+            paginationDiv.querySelector(".pagination-previous").removeAttribute("hidden")
+            if (page == 0) {
+                paginationDiv.querySelector(".pagination-previous").setAttribute("hidden","hidden")
+            }
+            paginationDiv.querySelector(".pagination-next").removeAttribute("hidden")
+            if (page == totalPages - 1) {
+                paginationDiv.querySelector(".pagination-next").setAttribute("hidden","hidden")
+            }
+
+            // Display at most "toBeDisplayed" page links in the paginator
+            const toBeDisplayed = 10
+            let [start, end] = setStartEndForPaginator(page, totalPages, toBeDisplayed)
```

Comment:
In the interest of simplicity I’d prefer we always display one page link per page.

Reply:
I don't think that'll look or will be handy for others as its documentation, its long, and there will be many search results for which having more than 1 page link per page will make it easier to navigate. If 10 appears excessive then how about 5 or 7?

Reply:
The problem I see with the current implementation is that there’s no way to know how many pages there are except starting to navigate. Could we do what I suggest, and maybe additionally change our `hitsPerPage` on line 48 to return 100 results per page?